swarmlib/__main__.py

```diff
@@ -58,6 +58,10 @@ def run_swarm():
         default=None,
         type=int,
         help='Used to set the initial state of the random bit generator (default None)')
+    parser.add_argument(
+        '--log-level',
+        default="INFO",
```

Comment:
Can you make this a [choice](https://docs.python.org/3/library/argparse.html#choices) of `['debug', 'info', 'warning', 'error', 'critical']` ? Then the parser automatically deals with the error handling.